tests/test_populus_project.py

```diff
@@ -0,0 +1,18 @@
+from os.path import join, dirname, abspath
+
+import nkms_eth
+
+
+def test_testerchain_create(testerchain):
+    assert testerchain._network == 'tester'
+    assert testerchain._chain.web3.eth.blockNumber >= 0
+
+
+def test_nucypher_populus_project(testerchain):
+    project_dir = join(dirname(abspath(nkms_eth.__file__)), 'project')
+
+    assert testerchain._populus_config._project_dir == project_dir           # blockchain instance
+    assert testerchain._project.project_dir == project_dir                   # populus project
+
+    # raw compiled contract access
+    assert 'NuCypherKMSToken' in testerchain._project.compiled_contract_data
```

Comment:
Not immediately evident - why do I expect this string to appear?  And what I do expect the other parts of the string to be?

Reply:
This is a giant dictionary, mapping all compiled solidity code - In fact all identifiers can be found here.  Since we have a smart contract directive using that Identifier, it's a top-level key.